packages-native/bottom-sheet/README.md

```diff
@@ -1,3 +1,88 @@
-# Bottom drawer
+## Bottom sheet
 
-Highly configurable widget to introduce bottom sheet behavior.
+The bottom sheet widget creates a set of options while blocking interaction with the rest of the screen or a draggable surface anchored to the bottom of the screen. There are two variations: Modal bottom sheet and Expanding bottom sheet.
```

Comment:
There are two variations of bottom sheet widget: a modal bottom sheet and an expanding bottom sheet.

Reply:
Done